studio/schemas/documents/sections/readyToInvest.js

```diff
@@ -0,0 +1,41 @@
+import { SplitHorizontalIcon } from '@sanity/icons'
+import supportedLanguages from '../../supportedLanguages';
+
+const baseLanguage = supportedLanguages.find(l => l.isDefault);
+
+export default {
+  type: 'document',
+  name: 'readyToInvest',
+  title: 'Ready To Invest',
+  icon: SplitHorizontalIcon,
+  fields: [
+    {
+      name: 'heading',
+      type: 'localePortableText',
+      title: 'Heading (*)',
+      validation: Rule => Rule.error('Information required.').required(),
+    },
+    {
+      name: 'buttonText',
+      type: 'localePortableText',
```

Comment:
You're using a button, so it needs to be a `localeCta `in order to have a route. Otherwise the button won't redirect anywhere when clicked. Check an example in `web\components\sections\HeroDoubleButton\HeroDoubleButton.js`